var/spack/repos/builtin/packages/aocl-sparse/package.py

```diff
@@ -98,13 +98,11 @@ def cmake_args(self):
             args.append(self.define_from_variant("BUILD_ILP64", "ilp64"))
 
         if self.spec.satisfies("@4.0:"):
-            args.append("-DAOCL_BLIS_LIB={0}/libblis.so".format(self.spec["amdblis"].prefix.lib))
+            args.append("-DAOCL_BLIS_LIB=" + str(self.spec["blas"].libs))
```

Comment:
This is wrong, since the package doesn't have a dependency on `blas` (virtual package) - but on `blis` which is a specific implementation. I suggest reworking the properties in `blis` and `amdlibflame` so that:
```
spec["blis"].libs
```
returns the expected value.

Reply:
Suggestion changes are done.